src/localize/languages/pt-PT.json

```
@@ -0,0 +1,470 @@
+        "media_chunk_size": "Tamanho do ficheiro"
+      },
+      "profile": "Perfil",
+      "profiles": {
+        "high": "Alto",
+        "low": "Baixo"
+      },
+      "style": {
+        "border_radius": "Tamanho da borda",
+        "box_shadow": "Caixa de Fundo",
+        "editor_label": "Editor de rotulos"
+      },
+      "warning": "Avisos"
+    },
+    "view": {
+      "camera_select": "Visualização de câmeras recém-selecionadas",
+      "dark_mode": "Modo escuro",
+      "dark_modes": {
+        "auto": "Automático",
+        "off": "Desligado",
+        "on": "Ligado"
+      },
+      "default": "Visualização padrão",
+      "scan": {
+        "enabled": "Modo scan ativado",
+        "scan_mode": "Modo scan",
+        "show_trigger_status": "Pulsar borda quando acionado",
+        "untrigger_reset": "Redefinir a visualização para o padrão após desacionar",
+        "untrigger_seconds": "Segundos após a mudar para o estado inativo para desacionar"
+      },
+      "timeout_seconds": "Redefinir para a visualização padrão X segundos após a ação do usuário (0 = nunca)",
+      "update_cycle_camera": "Percorrer as câmeras quando a visualização padrão for atualizada",
+      "update_force": "Forçar atualizações do cartão (ignore a interação do usuário)",
+      "update_seconds": "Atualize a visualização padrão a cada X segundos (0 = nunca)",
+      "views": {
+        "clip": "Clipe mais recente",
+        "clips": "Galeria de clipes",
+        "current": "Visualização atual",
+        "image": "Imagem estática",
+        "live": "Visualização ao vivo",
+        "snapshot": "Snapshot mais recente",
+        "snapshots": "Galeria de Snapshots",
+        "timeline": "Visualização da linha do tempo"
+      }
+    }
+  },
+  "editor": {
+    "add_new_camera": "Adicionar nova câmera",
+    "button": "Botão",
+    "camera": "Câmera",
+    "cameras": "Câmeras",
+    "cameras_secondary": "Quais câmeras renderizar neste cartão",
+    "delete": "Excluir",
+    "dimensions": "Dimensões",
+    "dimensions_secondary": "Dimensões e opções de forma",
+    "image": "Imagem",
+    "image_secondary": "Opções de visualização de imagem estática",
+    "live": "Ao vivo",
+    "live_secondary": "Opções de visualização da câmera ao vivo",
+    "media_gallery": "Galeria",
+    "media_gallery_secondary": "Galeria Secundaria",
+    "media_viewer": "Visualizador de eventos",
+    "media_viewer_secondary": "Opções do visualizador de Snapshots e clipes",
+    "menu": "Menu",
+    "menu_secondary": "Opções de aparência do menu",
+    "move_down": "Descer",
+    "move_up": "Subir",
+    "overrides": "As substituições estão ativas",
+    "overrides_secondary": "Substituições de configuração dinâmica detectadas",
+    "timeline": "Linha do tempo",
+    "timeline_secondary": "Opções do evento da linha do tempo",
+    "upgrade": "Upgrade",
+    "upgrade_available": "Um upgrade automático da configuração de cartão está disponível",
```

meaningless phrase in portuguese
```suggestion
    "upgrade_available": "Está disponível uma atualização automática do cartão",
```